modules/timeline/src/main/Entry.scala

```diff
@@ -38,6 +38,7 @@ object Entry:
       case d: TeamJoin      => "team-join"       -> toBson(d)
```

Comment:
also adds to timeline - not a lot, but it's one of all the little additions that make a big one